test/ios-entitlements-service.ts

```diff
@@ -47,7 +47,7 @@ describe("IOSEntitlements Service Tests", () => {
 		injector = createTestInjector();
 
 		platformsData = injector.resolve("platformsData");
-		projectData = injector.resolve("projectData");
+		projectData = $injector.resolve<IProjectData>("projectData");
```

Comment:
this way you'll not use the local `injector` variable, but the global `$injector` one. It is a bad practice to use the global variable in the tests. Can you elaborate why you needed to change this?

Reply:
probably a copy-paste error, but thanks for the explanation